make/webconsole.mk

```diff
@@ -9,6 +9,6 @@ TARGET_DIR = $(PWD)/pkg/consoleplugin/contentserver/static
 generate-webconsole-scripts:
 	mkdir -p $(DIST_DIR)
 	rm -f $(DIST_DIR)/*
-	cd $(WEBCONSOLE_SRC_DIR);${IMAGE_BUILDER} build --no-cache --volume $(DIST_DIR):/dist:z,U -f $(WEBCONSOLE_SRC_DIR)/Containerfile .
+	cd $(WEBCONSOLE_SRC_DIR);${IMAGE_BUILDER} build --no-cache --volume $(DIST_DIR):/opt/app-root/dist:U -f $(WEBCONSOLE_SRC_DIR)/Containerfile .
```

Comment:
@sbryzak I removed the `z` flag since it causes troubles on macOS

```
STEP 5/15: RUN yum -y module enable nodejs:$NODEJS_VERSION &&     MODULE_DEPS="make gcc gcc-c++ libatomic_ops git openssl-devel" &&     INSTALL_PKGS="$MODULE_DEPS nodejs npm nodejs-nodemon nss_wrapper" &&     ln -s /usr/lib/node_modules/nodemon/bin/nodemon.js /usr/bin/nodemon &&     ln -s /usr/libexec/platform-python /usr/bin/python3 &&     yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS &&     rpm -V $INSTALL_PKGS &&     node -v | grep -qe "^v$NODEJS_VERSION\." && echo "Found VERSION $NODEJS_VERSION" &&     yum -y clean all --enablerepo='*'
Error: building at STEP "RUN yum -y module enable nodejs:$NODEJS_VERSION &&     MODULE_DEPS="make gcc gcc-c++ libatomic_ops git openssl-devel" &&     INSTALL_PKGS="$MODULE_DEPS nodejs npm nodejs-nodemon nss_wrapper" &&     ln -s /usr/lib/node_modules/nodemon/bin/nodemon.js /usr/bin/nodemon &&     ln -s /usr/libexec/platform-python /usr/bin/python3 &&     yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS &&     rpm -V $INSTALL_PKGS &&     node -v | grep -qe "^v$NODEJS_VERSION\." && echo "Found VERSION $NODEJS_VERSION" &&     yum -y clean all --enablerepo='*'": resolving mountpoints for container "ea81309972f94d5bfac479a5b79914ce65eadfc7c069ad3d46f6e5ecd990c762": lsetxattr /Users/xcoulon/code/go/src/github.com/codeready-toolchain/member-operator/webconsole_source/dist: operation not supported
```
please let me know if it's needed on your machine or if we can keep it removed